composer.json

```diff
@@ -0,0 +1,56 @@
+{
+    "name": "immediate/im-identity-api-bundle",
+    "description": "Communicating with Identity APIs",
+    "type": "symfony-bundle",
+    "license": "Proprietary",
+    "require": {
+        "php": "^8.0",
+        "league/container": "^4.0",
+        "php-http/client-common": "^2.4",
+        "php-http/discovery": "^1.19.0",
+        "psr/http-client": "^1.0",
+        "psr/http-factory": "^1.0",
+        "symfony/cache": "^6.0",
+        "symfony/dependency-injection": "^6.0",
+        "symfony/http-kernel": "^6.0"
+    },
+    "require-dev": {
+        "10up/wp_mock": "0.4.2",
```

Comment:
1.0 plx

Reply:
Technically not needed at all so I'll just remove outright.